SabreSprings.Brewing.Database/FermentationLog.sql

```diff
@@ -1,11 +1,12 @@
 ﻿CREATE TABLE [dbo].[FermentationLog]
```

Comment:
[dbo] is SQL Server, not sqlite

Reply:
Create table query changed to match SQLite